app/models/mixins/miq_web_server_worker_mixin.rb

```diff
@@ -15,7 +16,7 @@ def self.binding_address
 
     def self.rails_server_command_line
       @rails_server_command_line ||= begin
-        "#{self.nice_prefix} #{Rails.root.join("bin", "rails")} server".freeze
+        "#{nice_prefix} #{Rails.root.join("bin", "rails")} server #{RAILS_SERVER}".freeze
```

Comment:
To add to that I think that would be cleaner to keep it this way so that we won't have locate additional references to puma if we switch out the server again in the future.